packages/database/src/migrations/20201014220808-AddLaosEocAggregateForecastMapOverlays-modifies-data.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+import { cloneDeep } from 'lodash'
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+
+const MAP_OVERLAY_GROUP_CODE = 'LA_EOC_Weather_Observations_And_Forecast';
+
+const commonMapOverlayConfig = {
+  userGroup: 'Laos EOC User',
+  dataElementCode: 'WTHR_FORECAST_PRECIP',
+  measureBuilderConfig: {
+    entityAggregation: {
+      dataSourceEntityType: 'city',
+    },
+  },
+  measureBuilder: 'sumAllPerOrgUnit',
+  presentationOptions: {
+    scaleType: 'neutral',
+    valueType: 'oneDecimalPlace',
+    displayType: 'spectrum',
+    measureLevel: 'City',
+    periodGranularity: 'day',
+    timePeriodEditable: false,
```

Comment:
This prevents the user from changing the dates

Reply:
Oh sorry man I forgot to comment this: could we also rename this variable to `canSelectDateRange` or similar?

I find the `is...`, `.are...`, `can...` prefixes very useful for boolean options/vars

Reply:
Good point! Updated